### PR TITLE
Add tests sealed secrets to prod

### PR DESCRIPTION
**Pull request by luigibk**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/prod/common/bar/tests-values.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: bitnami.com/v1alpha1
```
Comment (timja) [inline k8s/prod/common/bar/tests-values.yaml]: prob want to update codeowners for this file pattern

Comment (luigibk) [reply k8s/prod/common/bar/tests-values.yaml]: Yeah, even if I suspect it will have to be us anyway :-)

Comment (timja) [reply k8s/prod/common/bar/tests-values.yaml]: i mean add us

File: CODEOWNERS [hunk 2]
```diff
@@ -49,16 +52,21 @@ k8s/aat/common/ccd/definition-store-api.yaml @hmcts/cdm
 k8s/aat/common/ccd/user-profile-api.yaml @hmcts/cdm
 k8s/perftest/common/ccd/user-profile-api.yaml @hmcts/cdm
 k8s/perftest/common/ccd/definition-store-api.yaml @hmcts/cdm
+k8s/aat/common/ccd/tests-values.yaml @hmcts/cdm
```
Comment (timja) [inline CODEOWNERS]: can't we just set **/tests-values.yaml to platform-engineering?

**Pull request merged**